server/routes.go

```diff
@@ -231,7 +231,7 @@ func GenerateHandler(c *gin.Context) {
 			ch <- r
 		}
 
-		if err := loaded.runner.Predict(c.Request.Context(), req.Context, prompt, fn); err != nil {
+		if err := loaded.runner.Predict(c.Request.Context(), req.Context, prompt, req.Format, fn); err != nil {
```

Comment:
We should add a check (way at the start of the GenerateHandler) that `format` is a value we support (ie: json).